docs/only-allow-admins-to-create-new-streams-feature.md

```diff
@@ -0,0 +1,12 @@
+# Only allow admins to create new streams
+
+1. login with a Admin account.
+2. locate the cog
+(![cog](/static/images/help/cog.png)) on the main navigation bar and click on
+it to relase the dropdown menu.
+![Settings dropdown](/static/images/help/settings-dropdown-menu-administration.png)
+3. Select the Administration option. This will open the Administration page
+of your organization.
+4. Under Organization settings select the **Only admins may create streams** checkbox.
+![Organization settings](/static/images/help/administration-organization-settings.png)
+5. Click on **Save changes**.
```

Comment:
I think you have put this file in the wrong directory. Pretty much all of the docs that are being created right now are being put in `templates/zerver/help`.